### PR TITLE
release v0.10.0

### PR DESCRIPTION
**Pull request by vdhanan**
**Is this a bug fix or adding new feature?**
release commit

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ resources:
 
 priorityClassName: ""
 nodeSelector: {}
-tolerateAllTaints: true
+tolerateAllTaints: false
```
Comment (ayberk) [inline charts/aws-ebs-csi-driver/values.yaml:73]: Why do we have these changes? Release commit should only have the version update.

Comment (vdhanan) [reply charts/aws-ebs-csi-driver/values.yaml:73]: yeah, in retrospect, this should have been a separate commit. will split it out into two commits next time.

**Comment by AndyXiangLi** [comment]
I will merge this PR first. Like Varun mentioned, next time we should only include release related change.
/approve
/lgtm

**Pull request merged**